test/unittests/test_all.zig

```diff
@@ -1,4 +1,5 @@
 test "all" {
     _ = @import("kernel/test_vga.zig");
     _ = @import("kernel/test_tty.zig");
+    _ = @import("../../src/kernel/arch.zig");
```

Comment:
This file should only contain `test_*.zig` files, so you will need to create a test_paging.zig file which will import the paging file. Then import the test_paging.zig here.

Reply:
Ah yes, I mistakenly included this from my testing. Good spot